grass7/raster/r.in.srtm.region/r.in.srtm.region.py

```diff
@@ -93,10 +93,16 @@
 
 import os
 import atexit
-import urllib
-import urllib2
+try:
+    from urllib2 import urlopen, URLError, HTTPError
```

Comment:
This should replace all the urllib stuff:

    from six.moves.urllib import request as urllib2

Reply:
Line

    urllib.urlcleanup()

should be changed to 

    urllib2.urlcleanup()

(with the import above)